gmloader/libyoyo.h

```diff
@@ -63,9 +65,10 @@ typedef struct Gamepad {
 	double buttons[16];
 	double deadzone;
 	double axis[4];
+	double motors[1];
```

Comment:
If there are 2 motors, it should be this ?
```suggestion
	double motors[2];
```

Reply:
It's 0-based. 0 and 1.